config/postgres/values.yaml.gotmpl

```
@@ -24,6 +24,7 @@ initdbScripts:
      CREATE DATABASE reporter;
      CREATE DATABASE "fistful-magista";
      CREATE DATABASE fraudbusters;
+     CREATE DATABASE fraudbusters-management;
```

Постгря не умеет в создание без кавычек баз с тире в имени
```suggestion
     CREATE DATABASE "fraudbusters-management";
```